server/control-panel.md

```diff
@@ -1,69 +1,39 @@
 # Control Panels
 
-## cPanel
+This document describes how to install WordPress® in the cPanel & WHM and Plesk control panels using WP Toolkit.
 
-This tutorial provides step-by-step examples of creating a database and database user using the cPanel MySQL Database Wizard tool.
+## WP Toolkit 
 
-### Login to your site {#login-to-your-site}
+WP Toolkit is a single management interface that allows you to install, configure, and manage WordPress easily. WP Toolkit can install, configure, and manage WordPress versions 4.9 or later.
 
-Login to your hosting control panel (cPanel) with information provided by your host company.
+cPanel & WHM versions 102 and above install WP Toolkit by default. It is available in Plesk if the server administrator has installed the WP Toolkit extension. The WP Toolkit extension is free with the Web Pro and the Web Host Plesk editions and is available for a fee for the Web Admin edition.
```

Comment:
please remove any advertising sentences for the panels. If the tool is not available for free it shouldn't be covered in the documentation here.